### PR TITLE
add a command to see all current user's posts

### PR DESCRIPTION
**Pull request by arrizalamin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -171,6 +171,26 @@ function start(token, username){
...
+  vorpal
+  .command('me posts', 'See all posts created by current user')
+  .action(function(args, callback) {
+    me(token, function(response){
+      var posts = response.body.user.posts;
+
+      posts.forEach(function(post) {
+        vorpal.log('\n  ' + chalk.bold.blue(post.name));
+        vorpal.log('  ' + chalk.italic(post.tagline));
+        vorpal.log('  ' + chalk.dim('Votes') + ': ' + chalk.bold(post.votes_count) + '   ' + chalk.dim('Comments') + ': ' + chalk.bold(post.comments_count));
```
Comment (jayvdb) [inline index.js]: wow - reallly long line.

**Comment by jayvdb** [comment]
Please `git squash` your two commits into a single commit, and force push.

**Comment by jayvdb** [comment]
👍

**Comment by Kristories** [comment]
Thank you @arrizalamin @jayvdb 🍻

**Pull request merged**